### ISSUE TITLE
java.lang.IllegalStateException: Not a JSON Array: 

### ISSUE THREAD

**Issue by wuseal** [wuseal/JsonToKotlinClass#87]
Plugin version: 2.3.0
Exception Log:
```
java.lang.IllegalStateException: Not a JSON Array: {"chapterID":"978","chapterName":"4、《诗经》两首","rate":"56"}
	at com.google.gson.JsonElement.getAsJsonArray(JsonElement.java:107)
	at wu.seal.jsontokotlin.utils.ExtensionsKt.onlyOneSubArrayContainsElementAndAllObjectRecursive(Extensions.kt:138)
	at wu.seal.jsontokotlin.TargetJsonElement.getTargetJsonElementForGeneratingCode(TargetJsonElement.kt:33)
	at wu.seal.jsontokotlin.KotlinCodeMaker.<init>(KotlinCodeMaker.kt:34)
	at wu.seal.jsontokotlin.KotlinCodeMaker.appendCodeMember(KotlinCodeMaker.kt:136)
	at wu.seal.jsontokotlin.KotlinCodeMaker.makeKotlinData(KotlinCodeMaker.kt:46)
	at wu.seal.jsontokotlin.KotlinCodeMaker.appendCodeMember(KotlinCodeMaker.kt:136)
	at wu.seal.jsontokotlin.KotlinCodeMaker.makeKotlinData(KotlinCodeMaker.kt:46)
	at wu.seal.jsontokotlin.GenerateKotlinFileAction.doGenerateKotlinDataClassFileAction(GenerateKotlinFileAction.kt:93)
	at wu.seal.jsontokotlin.GenerateKotlinFileAction.actionPerformed(GenerateKotlinFileAction.kt:67)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.lambda$actionPerformed$0(ActionMenuItem.java:301)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:307)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:104)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:291)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$0(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$submitTransaction$1(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:120)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:122)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:528)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.access$300(BegMenuItemUI.java:48)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:548)
	at java.awt.Component.processMouseEvent(Component.java:6548)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6313)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4526)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:822)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:759)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:423)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

From JSON:
```json
{
  "version": "语文-人教版-必修2",
  "subjectID": "12",
  "chapterList": [
    {
      "chapterID": "973",
      "chapterName": "第一单元",
      "rate": "56",
      "sub": [
        {
          "chapterID": "974",
          "chapterName": "1、荷塘月色",
          "rate": "56"
        },
        {
          "chapterID": "975",
          "chapterName": "2、故都的秋",
          "rate": "56"
        }
      ]
    },
    {
      "chapterID": "977",
      "chapterName": "第二单元",
      "rate": "56",
      "vip": 1,
      "sub": [
        {
          "chapterID": "978",
          "chapterName": "4、《诗经》两首",
          "rate": "56"
        },
        {
          "chapterID": "979",
          "chapterName": "5、离骚",
          "rate": "56"
        },
        null
      ]
    }
  ]
}
```

**Comment by wuseal** [comment]
This JSON Will break again:
```JSON
{
  "version": "语文-人教版-必修2",
  "subjectID": "12",
  "chapterList": [
    {
      "chapterID": "973",
      "chapterName": "第一单元",
      "rate": "56",
      "sub": [
        {
          "chapterID": "974",
          "chapterName": "1、荷塘月色",
          "rate": "56"
        },
        {
          "chapterID": "975",
          "chapterName": "2、故都的秋",
          "rate": "56"
        }
      ]
    },
    {
      "chapterID": "977",
      "chapterName": "第二单元",
      "rate": "56",
      "vip": 1,
      "sub": [
        {
          "chapterID": "978",
          "chapterName": "4、《诗经》两首",
          "rate": "56"
        },
        {
          "chapterID": "979",
          "chapterName": "5、离骚",
          "rate": "56"
        },
        null
      ]
    }
  ]
}
```

**Comment by wuseal** [comment]
Fixed In version 3.0.1
